specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -1984,6 +2272,88 @@
         }
       }
     },
+    "SqlServerInstanceProperties": {
+      "description": "Properties of SqlServerInstance.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "null"
+        },
+        "edition": {
```

Comment:
>"edition": { [](start = 8, length = 12)

can version and edition be string enums?

Reply:
It can be string enums. Currently all existing resource types of thie resource provider define its property type as string. To support other types of property, an infra update is needed. Considering this resource type is for private preview, can we ask for an exception to keep it as string for now?